development/build/static.js

```diff
@@ -108,7 +110,7 @@ module.exports = function createStaticAssetTasks({
   }
 };
 
-function getCopyTargets(shouldIncludeLockdown, shouldIncludeSnow) {
+function getCopyTargets(shouldIncludeLockdown, shouldIncludeSnow, includePPOM) {
```

Comment:
Rather than adding a boolean for a single feature, would it be more scalable to pass the `activeFeatures` array here so we can add further conditions to these targets in future without more arguments?

Reply:
Done